examples/falcon_cleanup_hosts/main.go

```diff
@@ -21,6 +21,7 @@ func main() {
 	clientCloud := flag.String("cloud", os.Getenv("FALCON_CLOUD"), "Falcon cloud abbreviation (us-1, us-2, eu-1, us-gov-1)")
 	dryRun := flag.Bool("dry-run", true, "Dry run will not remove the devices from Falcon")
 	filter := flag.String("filter", "", "Host search expression using Falcon Query Language (FQL)")
+	limit := flag.Int("limit", 25, "The number of entries to handle at a time. Maximum limit of 100")
```

Comment:
What would you say about `--batch-size` ?

Reply:
Done